spec/fixtures/model/force/lease/lease_salesforce_from_domain.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
does this file name need the "from_domain" or is it just lease_salesforce?

Reply:
we specify both the origin format and the result format, because the content of a translated object will depend on where that object originated from. objects can have fields that are only present when they are fetched from salesforce, or fields that are only present when they are fetched from the custom api. if i fetch, say, an application from the custom api, and then translate it to domain, it will only have the domain correlates of the custom api fields. if i fetch that same application via SOQL and translate it to domain, it will have the domain field correlates for those salesforce fields, which are a slightly different set of fields from what is returned by the custom api. this is a lot more relevant for application and preference objects, but i wanted to maintain consistency when testing all objects.